ui/src/common/time.ts

```diff
@@ -211,4 +236,8 @@ export class TPTimeSpan implements Span<TPTime, TPDuration> {
   pad(padding: TPDuration): Span<TPTime, TPDuration> {
     return new TPTimeSpan(this.start - padding, this.end + padding);
   }
+
+  toString(): string {
+    return `TPTimeSpan(${this.start}, ${this.end})`;
```

Comment:
```suggestion
    return `${TPTimeSpan.name}(${this.start}, ${this.end})`;
```

Would also allow it to be refactored.

Reply:
What would it mean for minimization that involves name-mangling? Maybe something that is more generic like just "Time" would be more appropriate, anyways.

Reply:
True, thanks, I did not mind that. It depends on the compiler/minifier. This can be resolved.